src/intTest/resources/edifact/multi_pathology_msg2.fhir.json

```diff
@@ -59,6 +59,23 @@
         ]
       }
     },
+    {
```

Comment:
I think the `subject` field containing a reference (the `id` from the `Patient` resource) is missing from the `multi_pathology.fhir` JSON messages for the `ProcedureRequest` resources

Reply:
Here we go. Thanks!
6753168f1d300192299892f1dd8452e8f6062a37